internal/cli/command/daemon/agent.go

```diff
@@ -75,41 +75,33 @@ func SetAgentConfigFromEnvs(cfg *config.Config) error {
 	if apiAddress != "" {
 		cfg.API.Address = apiAddress
 	}
-	apiTlsVerify := os.Getenv("LB_API_TLS_VERIFY")
-	if apiTlsVerify != "" {
-		cfg.API.TLS.Verify = converter.StringToBool(apiTlsVerify)
+	apiTLSVerify := os.Getenv("LB_API_TLS_VERIFY")
+	if apiTLSVerify != "" {
+		cfg.API.TLS.Verify = converter.StringToBool(apiTLSVerify)
 	}
-	apiTlsCa := os.Getenv("LB_API_TLS_CA_FILE")
-	if apiTlsCa != "" {
-		cfg.API.TLS.FileCA = apiTlsCa
+	apiTLSCa := os.Getenv("LB_API_TLS_CA_FILE")
+	if apiTLSCa != "" {
+		cfg.API.TLS.FileCA = apiTLSCa
 	}
-	apiTlsCert := os.Getenv("LB_API_TLS_CERT_FILE")
-	if apiTlsCa != "" {
-		cfg.API.TLS.FileCert = apiTlsCert
+	apiTLSCert := os.Getenv("LB_API_TLS_CERT_FILE")
+	if apiTLSCa != "" {
```

Comment:
Looks like you use incorrect variable. I think `apiTLSCert` should be instead of `apiTLSCa`.